extensions/wikia/Search/classes/Result.php

```diff
@@ -64,78 +76,91 @@ public function getText( $field = 'text', $wordLimit = null ) {
 		$textAsString = static::limitTextLength( $textAsString, $wordLimit );
 
 		// if title and description both start with the same File: prefix, remove the prefix from description
-		$textAsString = $this->removePrefix($this->findFilePrefix($this->getTitle(false)), $textAsString);
+		$textAsString = $this->removePrefix( $this->findFilePrefix( $this->getTitle( false ) ), $textAsString );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$textAsString". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)